### PR TITLE
Catalog macb fix

### PR DESCRIPTION
**Pull request by steph-lieu**
Add optional configuration to enable IMS managed ACBs and registration of IMS Catalog to DBRC.
This was tested using zos_core collection 1.5.0 and ZOAU 1.2.4 with all combinations of the variable representing the enablement of IMS managed ACBs and registration of IMS Catalog to DBRC. 

-     CATALOG_DBRC_ENABLED - True ; MANAGED_ACB_ENABLED - False
-     CATALOG_DBRC_ENABLED - False ; MANAGED_ACB_ENABLED - True
-     CATALOG_DBRC_ENABLED - False ; MANAGED_ACB_ENABLED - False
-     CATALOG_DBRC_ENABLED- True ; MANAGED_ACB_ENABLED - True

### CODE REVIEW

**Review by jashnani — CHANGES_REQUESTED** [review]
Looks good overall. Let's add a brief summary of the testing performed as part of this effort to the PR description for reference. Have a few questions and suggestions below.

File: zos_subsystems/ims/ims_provisioning/README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ playbook notes sections for additional details and configuration.
 
 This project uses roles to provide an object-oriented model to provision IMS.  Each role is responsible for a specific area.  These roles can be re-used in different playbooks.
 
+- [**image_copy**](roles/image_copy/README.md) - creates and deletes image copies of IMS catalog
```
[the remark this answers is not in the document]
Comment (steph-lieu) [reply zos_subsystems/ims/ims_provisioning/README.md:25]: Yes -- I have renamed it to `ims_catalog_image_copy`

File: zos_subsystems/ims/ims_provisioning/roles/image_copy/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+=========
+
+Create [image copies](https://www.ibm.com/docs/en/ims/15.3.0?topic=copies-image-ims-image-copy-utilities) by using the [Database Image Copy utility (DFSUDMP0)](https://www.ibm.com/docs/en/ims/15.3.0?topic=utilities-database-image-copy-utility-dfsudmp0) of the loaded IMS catalog data sets. You need image copies when catalog is DBRC enabled.
+
+Requirements
+------------
+
+[IBM&reg; z/OS&reg; core collection 1.6.0 or later](https://galaxy.ansible.com/ibm/ibm_zos_core)
+
+[IBM&reg; z/OS&reg; IMS collection 1.3.0-beta.1](https://galaxy.ansible.com/ibm/ibm_zos_ims)
```
[the remark this answers is not in the document]
Comment (steph-lieu) [reply zos_subsystems/ims/ims_provisioning/roles/image_copy/README.md:11]: Changed -- also changed the core collection to 1.5.0 or later

File: zos_subsystems/ims/ims_provisioning/ansible.cfg [hunk 3]
```diff
@@ -48,8 +48,8 @@ strategy = free
 var_compression_level = 9
 
 [ssh_connection]
-ssh_args = -C -o ControlMaster=auto -o ControlPersist=1200s -o PreferredAuthentications=publickey -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null
+#ssh_args = -C -o ControlMaster=auto -o ControlPersist=1200s -o PreferredAuthentications=publickey -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null
```
[the remark this answers is not in the document]
Comment (steph-lieu) [reply zos_subsystems/ims/ims_provisioning/ansible.cfg:51]: Tested with it uncommented and it still works. Originally uncommented to try a workaround for core 1.5.0, that didn't end up working out.

File: zos_subsystems/ims/ims_provisioning/roles/ims_common/tasks/main.yml [hunk 4]
```diff
@@ -44,7 +44,8 @@
         src: IMSTESTL.IMS1.JOBS(IMS1HWS1)
         location: DATA_SET
         wait: true
-        wait_time_s: 30
+        wait_time_s: 60
+      ignore_errors: true
```
[the remark this answers is not in the document]
Comment (steph-lieu) [reply zos_subsystems/ims/ims_provisioning/roles/ims_common/tasks/main.yml]: Changed to back to 30 seconds and added comments to explain why we are ignoring errors.
Was originally 60 to experiment and test changes for the error related to the IMS Connect.

**Review by jashnani — APPROVED** [review]
Thanks for addressing the feedback. Looks good!

**Pull request merged**